setup.py

```diff
@@ -15,17 +15,10 @@
 
 """Aexpect setup script"""
 
-import sys
-# pylint: disable=E0611
-
 from setuptools import setup
 
 
 if __name__ == '__main__':
-    if sys.version_info[0] == 2:
-        REQUIREMENTS = ['subprocess32>=3.2.6']
```

Comment:
Looks we need to modify the following part accordingly?

https://github.com/avocado-framework/aexpect/blob/be9be4b325ac1da7b0c908e82f1d2c52e43dfd2f/aexpect/client.py#L55-L58